src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+import Icon from '~core/Icon';
+import Heading from '~core/Heading';
+import HookedUserAvatar from '~users/HookedUserAvatar';
+import { Colony } from '~data/index';
+import { Address } from '~types/index';
+import { getMainClasses } from '~utils/css';
+
+import useAvatarDisplayCounter from '../../../hooks/useAvatarDisplayCounter';
+
+import styles from './VoteResultsWidget.css';
+
+interface Appearance {
+  theme?: 'approve' | 'disapprove';
+}
+
+interface Props {
+  colony: Colony;
+  appearance?: Appearance;
+  value?: number;
+  title: string;
+  voters?: Address[];
+  maxAvatars?: number;
+}
+
+const displayName = 'VoteResultsWidget';
+
+const VoteResultsWidget = ({
+  colony: { colonyAddress },
+  appearance = { theme: 'approve' },
+  value = 0,
+  title,
+  voters = [],
+  maxAvatars = 3,
```

Comment:
I think this should actually be `minAvatars`, so that we can configure the minum number of avatars to display

Reply:
If it was `minAvatars` instead of `maxAvatars`, then that means that if I do something like `minAvatars={3}` I would expect that the widget shows **at least** 3 or more avatars? So if we have 15 voters, it would show the 15 avatars?

Reply:
Yes, you are right. You can ignore this.